### ISSUE TITLE
unwrapInline currently broken

### ISSUE THREAD

**Issue by tyler-johnson** [ianstormtaylor/slate#409]
Calling `unwrapInline()` throws the following error. This can be seen in the [links example](http://slatejs.org/#/links).

```
Uncaught Error: Invalid `key` argument! It must be either a block, an inline, a text, or a string. You passed: "null".(…)
```

This is not the first time this error has come up, it has [surfaced](/ianstormtaylor/slate/issues/365) [several](/ianstormtaylor/slate/issues/366) [times](/ianstormtaylor/slate/issues/370) over the last few weeks. It seems that this all comes down to the discussion in #388, which was fixed in #389. However, that patch was closed in favor of #396 which (hopefully) fixes this problem for good.

So what is the time-table for #396? If it isn't within the next week, would it possible to get #389 merged in the meantime? This bug is a major hold up for me as it puts my editor in a broken state and will require me to fork to get things working.

**Comment by khrykin** [comment]
@tyler-johnson Why is this closed? Seems like links example is still broken.

**Comment by tyler-johnson** [comment]
Well I closed it as @SamyPesse sort of answered my question here: https://github.com/ianstormtaylor/slate/pull/396#issuecomment-256861705

The changes were supposed to be done last week but I guess it is taking longer than expected. This will get fixed so I figured I wouldn't pollute the project with unnecessary issues.

If this is a critical issue for you, my suggestion is to fork the project temporarily with #389 which is what I have done.
